### PR TITLE
Repo readme

### PR DESCRIPTION
**Pull request by linahKhayri**
---
name: solution review 
about: A template PR for code review with a checklist
---

### Files

- [ ] The file name describes the function's behavior
- [ ] There is a module docstring in the function file
- [ ] The test file's name matches the function file name -
  `/tests/test_file_name.py`
- [ ] There is a module docstring in the tests file

### Unit Tests

- [ ] The test class has a helpful name in PascalCase
- [ ] The test class has a docstring
- Every unit test has
  - [ ] A helpful name
  - [ ] A clear docstring
  - [ ] Only one assertion
  - [ ] There is no logic in the unit test
- [ ] All tests pass
- [ ] There are tests for defensive assertions
- [ ] There are tests for boundary cases

### Function Docstring

- [ ] The function's behavior is described
- The function's arguments are described:
  - [ ] Type
  - [ ] Purpose
  - [ ] Other assumptions (eg. if it's a number, what's the expected range?)
- The return value is described
  - [ ] Type
  - [ ] Other assumptions are documented
- The defensive assertions are documented using `Raises:`
  - [ ] Each assumption about an argument is checked with an assertion
  - [ ] Each assertion checks for _only one_ assumption about the argument
- [ ] Include 3 or more (passing!) doctests

### The Function

- [ ] The function's name describes it's behavior
- [ ] The function's name matches the file name
- [ ] The function has correct type annotations

### Do's

- [ ] Variable names help to understand the strategy
- [ ] Any comments are clear and describe the strategy
- [ ] Lines of code are spaced to help show different stages of the strategy

### Don'ts

- [ ] The function's strategy _is not_ described in the documentation
- [ ] Comments explain the _strategy_, **not** the _implementation_
- [ ] The function _does not_ have more comments than code
  - If it does, consider finding a new strategy or a simpler implementation

## Implementation

- [ ] The code passes the formatting checks
- [ ] The code passes all Ruff linting checks
- [ ] The code has no (reasonable) Pylint errors
  - In code review, you can decide when fixing a Pylint error is helpful and
    when it's too restricting.
- [ ] Variables are named with snake_case
- [ ] Variable names are clear and helpful
- [ ] The code follows the strategy as simply as possible
- [ ] The implementation is as simple as possible given the strategy
- [ ] The code includes defensive assertions
- [ ] Defensive assertions include as little logic as possible

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- [Our Retrospectives](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-24/blob/main/collaboration/retrospective.md)
+- [Our challenges](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-24/tree/main/solutions)
+
+---
+
+## Meet the Dream Team 🧑‍🤝‍🧑
+
+Our squad is made up of some seriously talented people who bring the 🔥 to this project. Check out the crew making it all come to life:
+
+- **Abel Mesfin Teka**
```
Comment (robiel0143) [inline README.md:42]: a student in Computer Science

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- **Banchiamlak**
```
Comment (robiel0143) [inline README.md:43]: a medical doctor graduate

File: README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- **Gai Samuel**
```
Comment (robiel0143) [inline README.md:44]: a student

File: README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- **Khusro Sakhi**
```
Comment (robiel0143) [inline README.md:45]: I don't know khusros but you can ask him , i think he is a student at computer science

File: README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- **Linah Khayri**
```
Comment (robiel0143) [inline README.md:46]: you know what you are learning...

File: README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- **Mohamed Elnageeb**
```
Comment (robiel0143) [inline README.md:47]: a student at Mecha-electronics who just finished this year

File: README.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+- **Robel Mengsteab**
```
Comment (robiel0143) [inline README.md:48]: an electrical and electronics engineer

File: README.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+- **Viktoriya Haiduk**
```
Comment (robiel0143) [inline README.md:49]: a mathematician graduate

File: README.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+- **Yonatan Yishkl Yifat**
```
Comment (robiel0143) [inline README.md:50]: you can ask him as well

File: README.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+- **Zeinab Mohammed**
```
Comment (robiel0143) [inline README.md:51]: a chemical Engineer

File: README.md (hunk 1 above)
Comment (TekaMesfinAbel) [reply README.md:42]: Just a student, don't have a major yet. still a freshman.

File: README.md (hunk 2 above)
Comment (TekaMesfinAbel) [reply README.md:43]: Banchiamlak last name is Tessema

File: README.md (hunk 6 above)
Comment (Mohamed-Elnageeb) [reply README.md:47]: yeah you can almost say a Mechatronics engineer now.  😂😂😅

**Comment by linahKhayri** [comment]
@robiel0143 Hey Robel! 
amazing feedback and I LOVE your ideas, but I linked the github profile to our name as Khusro suggested so now when pressing the names it will lead to our profiles.
I think this way saves us from having to specify everyone's profession since it'll still provide the all these info in their profiles, but what do you think?

**Comment by Mohamed-Elnageeb** [comment]
yeah i agree makes it more concise and professional

**Review by Mohamed-Elnageeb — DISMISSED** [review]
looks good to me.

**Comment by TekaMesfinAbel** [comment]
Yeah that is better.

**Review by TekaMesfinAbel — DISMISSED** [review]

**Review by GaiSamuel — APPROVED** [review]
looks good!

**Review by Mohamed-Elnageeb — APPROVED** [review]
this hasn't been merged yet ? 😅

**Review by TekaMesfinAbel — APPROVED** [review]

**Comment by TekaMesfinAbel** [comment]
We can't merge it now. there is a comment made by Robel that needs to be resolved first, and I think we can't resolve that as long as his account is suspended.

**Review by Zeinab15 — APPROVED** [review]
Great job 👌
